lib/split/configuration.rb

```diff
@@ -25,6 +25,7 @@ class Configuration
     attr_accessor :beta_probability_simulations
     attr_accessor :winning_alternative_recalculation_interval
     attr_accessor :redis
+    attr_accessor :dashboard_pagination_default_per
```

Comment:
Can you rename this to `dashboard_pagination_default_per_page`?

Without any context, it's easier to understand that than 'pagination_per'.